<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoSignUpDialog</class>
 <widget class="QDialog" name="MoSignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>382</width>
     <height>237</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="ECRadioButton">
      <property name="text">
       <string>英/中</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="planLabel">
      <property name="text">
       <string>每日学习量</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="sexLabel">
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwdAgainLabel">
      <property name="text">
       <string>再次输入密码：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="passwdLabel">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QPushButton" name="signUpBtn">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QRadioButton" name="CERadioButton">
      <property name="text">
       <string>中/英</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="passwdAgainLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="quitBtn">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="maleRadioButton">
      <property name="text">
       <string>男</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QSpinBox" name="planSpinBox">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QRadioButton" name="femaleRadioButton">
      <property name="text">
       <string>女</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="memoryModeLabel">
      <property name="text">
       <string>记忆模式</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="userNameLineEdit"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="passwdLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
